public/src/ajaxify.js

```diff
@@ -424,17 +414,14 @@ $(document).ready(function () {
 		});
 	}
 
-	templates.registerLoader(ajaxify.loadTemplate);
+	require(['benchpress'], function (Benchpress) {
+		Benchpress.registerLoader(ajaxify.loadTemplate);
+	});
 
 	if (window.history && window.history.pushState) {
 		// Progressive Enhancement, ajaxify available only to modern browsers
 		ajaxifyAnchors();
 	}
 
 	app.load();
-
-	$('[type="text/tpl"][data-template]').each(function () {
```

Comment:
This was used to render 500.tpl when the server was down (ie loading the template from server will fail)

https://github.com/NodeBB/nodebb-theme-persona/blob/master/templates/footer.tpl#L5
https://github.com/NodeBB/NodeBB/blob/master/src/views/500-embed.tpl

Is there a replacement for it?